### ISSUE TITLE
Allow users to specify templates

### ISSUE THREAD

**Issue by ZedThree** [Fortran-FOSS-Programmers/ford#280]
One of the things I really love about Ford is that it looks great out-of-the-box. However, it would also be great if users could have a bit more control over the look. Having some mechanism for users to specify their own templates would be really cool.

**Comment by zbeekman** [comment]
I agree. I don't think this would be terribly hard to implement. The jinja2 templates are in their own subdirectory, and jinja2 supports inheritance. So presumably, we could add an option to the project file to tell ford that it should supplement the default templates with any user-defined ones.

**Comment by samhatfield** [comment]
Perhaps this would be a good opportunity to update to the latest version of Bootstrap as well?

**Comment by kevinhng86** [comment]
I will be able to rebuild the template entry point to enable ford from able to have as many templates as possible.
